lib/install/inflate-shrinkwrap.js

```diff
@@ -101,7 +109,15 @@ function inflatableChild (onDiskChild, name, topPath, tree, sw, requested, opts)
     sw.resolved = sw.version
     sw.version = regTarball
   }
-  if (sw.requires) Object.keys(sw.requires).map(_ => { sw.requires[_] = tarballToVersion(_, sw.requires[_]) || sw.requires[_] })
+  if (sw.requires) {
+    Object.keys(sw.requires).forEach(_ => {
```

Comment:
While we're in here making changes, can you change this to a more meaningful variable name?  Something like this:

```js
Object.keys(sw.requires).forEach(name => {
  const spec = sw.requires[name]
  sw.requires[name] = tarballToVersion(name, spec) ||
    relativizeLink(name, spec, topPath, requested) ||
    spec
})
```

Reply:
Thanks, applied the suggestion and changed the variable naming.